experiments/AMIP/modular/mpi/mpi_init.jl

```diff
@@ -4,7 +4,7 @@ using Logging
 
 const is_distributed = get(ENV, "CLIMACORE_DISTRIBUTED", "") == "MPI"
 if is_distributed
-    const comms_ctx = ClimaComms.context(ClimaComms.CPUDevice())
+    const comms_ctx = ClimaComms.context(ClimaComms.CPUSingleThreaded())
```

Comment:
This is fine for now, but I suggest looking at how ClimaAtmos does this-- this entire block should probably be fixed